analyzer/src/funTest/assets/projects/synthetic/pub/no-lockfile/.packages

```diff
@@ -0,0 +1,58 @@
+# Generated by pub on 2019-06-13 14:21:50.038631.
```

Comment:
This file should be removed from the commit. Also please add a .gitignore to this directory including this file and others that are generated when running the analyzer.

Reply:
done